src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/RootColorSliderUI.java

```diff
@@ -0,0 +1,62 @@
+package me.zeroeightsix.kami.gui.kami.theme.kami;
+
+import me.zeroeightsix.kami.gui.kami.RenderHelper;
+import me.zeroeightsix.kami.gui.kami.RootSmallFontRenderer;
+import me.zeroeightsix.kami.gui.kami.theme.kami.KamiGuiColors.GuiC;
+import me.zeroeightsix.kami.gui.rgui.component.container.Container;
+import me.zeroeightsix.kami.gui.rgui.component.use.ColorSlider;
+import me.zeroeightsix.kami.gui.rgui.render.AbstractComponentUI;
+import me.zeroeightsix.kami.gui.rgui.render.font.FontRenderer;
+import me.zeroeightsix.kami.util.HSBColourHolder;
+
+import static me.zeroeightsix.kami.util.ColourConverter.toF;
+import static org.lwjgl.opengl.GL11.*;
+
+/**
+ * Created by Guac on 10/8/2020.
+ */
+public class RootColorSliderUI extends AbstractComponentUI<ColorSlider> {
+
+    RootSmallFontRenderer smallFontRenderer = new RootSmallFontRenderer();
+
+    @Override
+    public void renderComponent(ColorSlider component, FontRenderer aa) {
+        glColor4f(toF(GuiC.sliderColour.color.getRed()), toF(GuiC.sliderColour.color.getGreen()), toF(GuiC.sliderColour.color.getBlue()), component.getOpacity());
+        glLineWidth(10f);
+        int width = component.getWidth();
+        int height = component.getHeight();
+        HSBColourHolder value = component.getValue();
+        double w = width * (value.getH());
+        float downscale = 2f;
+        glShadeModel(GL_SMOOTH);
```

Comment:
To draw smooth line, you should also do:
```suggestion
        glEnable(GL_LINE_SMOOTH);
        glHint(GL_LINE_SMOOTH_HINT, GL_NICEST);
```